mappings/net/minecraft/enchantment/Enchantment.mapping

```diff
@@ -1,17 +1,53 @@
 CLASS none/ahm net/minecraft/enchantment/Enchantment
+	CLASS none/ahm$a Weight
+		FIELD a COMMON Lnone/ahm$a;
+		FIELD b UNCOMMON Lnone/ahm$a;
+		FIELD c RARE Lnone/ahm$a;
+		FIELD d LEGENDARY Lnone/ahm$a;
+		FIELD e WEIGHT I
+		METHOD a getWeight ()I
+	FIELD a slotTypes [Lnone/sj;
 	FIELD b REGISTRY Lnone/db;
+	FIELD c type Lnone/ahn;
 	FIELD d translationName Ljava/lang/String;
+	FIELD e weight Lnone/ahm$a;
+	METHOD <init> (Lnone/ahm$a;Lnone/ahn;[Lnone/sj;)V
+		ARG 0 weight
+		ARG 1 type
+		ARG 2 slotTypes
 	METHOD a getTranslationKey ()Ljava/lang/String;
+	METHOD a getProtectionAmount (ILnone/rp;)I
+		ARG 0 level
+		ARG 1 source
+	METHOD a getAdditionalDamge (ILnone/ss;)F
+		ARG 0 level
+		ARG 1 entityType
+	METHOD a isAcceptableItem (Lnone/aeq;)Z
+		ARG 0 item
 	METHOD a isEqual (Lnone/ahm;)Z
 		ARG 0 other
+	METHOD a getEquipment (Lnone/sn;)Ljava/util/List;
+		ARG 0 entity
+	METHOD a onTargetDamaged (Lnone/sn;Lnone/se;I)V
+		ARG 0 user
+		ARG 1 target
+		ARG 2 level
+	METHOD b getHighestLevel ()I
 	METHOD b getEnchantmentById (Ljava/lang/String;)Lnone/ahm;
 		ARG 0 id
 	METHOD b getRawIdFromEnchantment (Lnone/ahm;)I
 		ARG 0 enchantment
+	METHOD b onUserDamaged (Lnone/sn;Lnone/se;I)V
```

Comment:
This method is for armor only right? If so, I concur with @copygirl, Wearer would make more sense.